### DISCUSSION TITLE
Useful R packages

### DISCUSSION THREAD

**Discussion by rosalieb** [JeromeMathieuEcology/club-iEES#21] [General]
_groundhog_ is an useful package to manage your libraries version: https://groundhogr.com/

From their website:

> Groundhog in a nutshell.
> 
> Make your R scripts reproducible by replacing `library(pkg)` with `groundhog.library(pkg,date)`
> 
> - `groundhog.library()` loads packages & their dependencies as available on chosen date on CRAN.  
> - Packages get automatically installed if needed.  
> - Installation keeps, rather than replaces, existing other versions of that package.  
> - All operating systems and R versions install/load the same package version  
> - If you want to discontinue using groundhog, simply:  groundhog.library(pkg,date).  
>

**Comment by rosalieb** [comment]
An useful thread about using colors as a tool in scientific presentations: https://twitter.com/iamscicomm/status/1531651977628770308

**Reply by sjijon** [reply]
Another nice article about color choice:

How to pick the least wrong colors: An algorithm for creating color palettes for data visualization

https://matthewstrom.com/writing/how-to-pick-the-least-wrong-colors
